### PR TITLE
feat(package): build es version compatible with node

### PR DESCRIPTION
**Pull request by davidyuk**
depends on #1369 (needed for cli)

So, after this would be possible to do:
```js
import { Crypto } from '@aeternity/aepp-sdk/es/index.mjs'

console.log(Crypto.generateKeyPair())
```
and execute it using plain node:
```
$ node test.mjs
{
  publicKey: 'ak_jnZsCzXTKdGDkdvSpExo46GwuWGv84CkzcvEtd6XdV98gLjd8',
  secretKey: 'cb8556c8ff170c0075c3a8bdfa0343c83782e1ccbca0e7842a0caaee92c10fce61266723868d53aab664039e4088d72df242a1510f2bab2e9c67dbeb768ac1ad'
}
$ node --version                                            
v16.13.0
```

### CODE REVIEW

**Review by subhod-i — APPROVED** [review]

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -4,19 +4,13 @@ jobs:
...
-      - if: github.ref != 'refs/heads/develop'
-        uses: actions/checkout@v2
-        with:
-          ref: develop
       - uses: actions/checkout@v2
         with:
-          fetch-depth: 100
-      - if: github.ref != 'refs/heads/develop'
-        run: git branch develop origin/develop
+          fetch-depth: 50
```
Comment (thepiwo) [inline .github/workflows/main.yml:9]: why do we need fetch-depth at all?

Comment (thepiwo) [reply .github/workflows/main.yml:9]: ah I see, to lint the last 50 commits

File: src/chain/index.js [hunk 2]
```diff
@@ -37,7 +37,27 @@ import { required } from '@stamp/required'
  * @return {Object} Chain instance
  */
 const Chain = stampit({
-  deepProps: { Ae: { defaults: { waitMined: true, verify: true } } }
+  deepProps: {
+    Ae: {
+      defaults: {
```
Comment (thepiwo) [inline src/chain/index.js:42]: why is this needed here?

File: src/chain/index.js [hunk 3]
```diff
@@ -37,7 +37,27 @@ import { required } from '@stamp/required'
...
+        waitMined: true,
+        verify: true,
+        _expectedMineRate: 180000,
+        _microBlockCycle: 3000,
+        _maxPollInterval: 5000
+      }
+    }
+  },
+  methods: {
+    _getPollInterval (type) {
```
Comment (thepiwo) [inline src/chain/index.js:52]: why is this needed here?

**Review by thepiwo — APPROVED** [review]

File: src/chain/index.js (hunk 3 above)
Comment (davidyuk) [reply src/chain/index.js]: so, these all are changes from #1369, I've rebased this PR on the top of develop and they are not visible in the diff anymore
